src/navigation-bar/navigation-bar.css

```diff
@@ -4,27 +4,39 @@
   line-height: 30px;
   background: white;
   display: flex;
+  border: 1px solid #e2e2e2;
 }
 
 .navigation-bar .buttons {
   flex: none;
 }
 
+.navigation-bar .buttons>button {
```

Comment:
Over specifying is bad. Maybe give the button a class which you could target with `.navigation-bar .go-up`?